spec/discard/model_spec.rb

```diff
@@ -81,9 +81,7 @@
 
       describe '#undiscard!' do
         it "raises Discard::RecordNotUndiscarded" do
-          expect {
```

Comment:
Sorry, I don't follow this. 

We don't expect Discard::RecordNotUndiscarded error anymore? Not sure why is that?